influxdb3_catalog/src/catalog.rs

```diff
@@ -843,10 +845,12 @@ mod tests {
                         "name": "db1",
                         "tables": {
                             "tbl1": {
+                                "table_id": 0,
```

Comment:
Should this start at 1?

Reply:
All of the Ids so far have started at 0.